.examples/apache/docker-compose.yml

```diff
@@ -1,20 +1,26 @@
-app:
-  image: matomo:3.5-apache
-  restart: always
-  links:
+version: '3'
+services:
+  app:
+    image: matomo:3.5-apache
+    restart: always
+    links:
     - db
-  volumes:
+    volumes:
     - ./config:/var/www/html/config:rw
     - ./logs:/var/www/html/logs
-  environment:
+    environment:
     - VIRTUAL_HOST=CHANGE_ME
+    ports:
+    - '80:80'
 
- db:
-  image: mariadb:latest
-  volumes:
+  db:
+    image: mariadb:latest
+    volumes:
     - ./mysql/runtime2:/var/lib/mysql
-  environment:
+    environment:
     - MYSQL_DATABASE=db
     - MYSQL_ROOT_PASSWORD=CHANGE_ME
     - MYSQL_USER=app
     - MYSQL_PASSWORD=CHANGE_ME
+    ports:
+    - '3306:3306'
```

Comment:
Why expose mysql ?

Reply:
@crazy-max Maybe you can open a PR to remove again these two lines?

Reply:
Done #119 ;)